### DISCUSSION TITLE
Can not open a exe file / is it the *.exe or another system issue?

### DISCUSSION THREAD

**Discussion by cxbignekoc** [x64dbg/x64dbg#2799] [Q&A]
Hi, this not an issue with x64dbg...its maybe the system or the *.exe...but i think it must be the system...???

```
My Sys:

CPU                 Intel(R) Core(TM)
i7-4790           CPU @ 3.60GHz 3.60 GHz
RAM               16,0 GB
Systemtype      64-Bit-Betriebssystem, x64 Prozessor
GFX                  Radeon R390 Series
Edition             Windows 10 Pro
Version            21H1
OS build        19043.1348
- MS Visual C++ 2015-2022 Redistributable -14.30.30704
- MS Visual C++ 2012
- MS Visual C++ 2013 (i use some older 32bit-Plugins in my Audio workstation)

```

Pre-Story.
I use VCV-Rack, a standalone modular audio system. - [https://vcvrack.com/]
i was running the previous version and its working well.
then i wanted to update and upgrade to the new Version V.2.0. I purchased the Pro Version then, because i like that thing.
I downloaded the installer, but i can not get any install process after accept the "unknown developer" and "changes on system".
Then tried to install the Free Version V2.0 i actually use, from inside the standalone App, but this not working too.

Want to check out the *.exe File, if maybe something is wrong, i have no clue what i am doing here. I am not a coder, just want infos.

Start the Debugger as Admin and open the exe-File (**RackPro-2.0.3b**). Then i get the Error:
- (0xc000007b)

I search the Web found a site and tried some workarounds
- uninstall older MS visual C++ redistributables
- install the latest one - MS Visual C++ 2015-2022 Redistributable -14.30.30704

, and i get no solve yet...after try install again.

Q: What ist the Problem?
Q: seems its not a general problem, so it is my or somebodys problem, but which one exactly?
Q: how can i solve this issue or how can i get more information?

Screenshot:
![error dbg](https://i.ibb.co/sszVh8L/vcv-dbg.png)
https://ibb.co/BTFCm08

**Comment by mrexodia** [comment]
0xC000007B STATUS_INVALID_IMAGE_FORMAT

This means your executable is corrupt or not compatible with your system, but I don’t think this is the right place to ask for tech support. Generally x64dbg doesn’t care and just calls CreateProcess.
